### PR TITLE
Update set method

### PR DESCRIPTION
**Pull request by gcasa**

### CODE REVIEW

File: Source/NSGraphicsContext.m [hunk 1]
```diff
@@ -515,6 +515,12 @@ + (void) handleExposeRect: (NSRect)rect forDriver: (void *)driver
 {
 }
 
+/* Private method for handling shadows */
+- (void) setShadow: (NSShadow *)shadow
+{
+  ASSIGN(_shadow, shadow);
```
Comment (fredkiefer) [inline Source/NSGraphicsContext.m:521]: Most likely a RELEASE in the dealloc method is needed

Comment (gcasa) [reply Source/NSGraphicsContext.m:522]: added.

**Comment by fredkiefer** [comment]
This PR now looks broken. It also includes the content of the last PR you already merged into master. Did your rebase go wrong or what happened here? The change itself is now correct but please try to sort out the history before merging.

**Comment by gcasa** [comment]
The rebases looked successful but I think I did something wrong.  I may restart the PR.

**Comment by gcasa** [comment]
Superseded by... https://github.com/gnustep/libs-gui/pull/233

**Pull request closed without merging**